docs/docs/configuration-parameters.md

```diff
@@ -93,7 +93,7 @@ The configured name will always be converted to a FQDN by ensuring it ends with
 `enforceRFC952` will enforce an older, more strict set of rules for DNS labels. For details, see the [RFC-952](https://tools.ietf.org/html/rfc952). The default value is `false`.
 
 `IPSources` defines a fallback list of IP sources for task records,
-sorted by priority. The default value is: `["netinfo", "mesos", "host"]`
```

Comment:
the default **coded** value hasn't changed (it's different than what's in config.json.sample, but that's not the same thing as "the default")